### DISCUSSION TITLE
[FEATURE] Use `.github/CODEOWNERS` data to show ownership in GH UI

### DISCUSSION THREAD

**Discussion by oleg-polivannyi** [community/community#8586] [Other Feature Feedback, Questions, & Ideas]
Examples:
- A file view in code review UI 
<img width="1252" alt="Screenshot 2021-12-09 at 13 46 26" src="https://user-images.githubusercontent.com/28644461/145400063-553e70ae-b2f2-4505-8083-efebf7ecb379.png">

- A file view in code exploration mode 
<img width="1647" alt="Screenshot 2021-12-09 at 13 49 56" src="https://user-images.githubusercontent.com/28644461/145400113-50b46675-36db-45ee-a086-b2d17a1ce9e1.png">

**Comment by sasha370** [comment]
It would be great to display сodowners for each changed file in the UI based on the data from `.github/CODEOWNERS`
